nengo_dl/tests/test_benchmarks.py

```diff
@@ -189,14 +189,8 @@ def test_lmu(Simulator, native_nengo, pytestconfig):
     [
         (benchmarks.cconv(128, 64, nengo.RectifiedLinear()), False, 64, 0.7, 0.85,),
         (benchmarks.cconv(128, 64, nengo.LIF()), False, 64, 1.5, 1.7),
-        (
-            benchmarks.integrator(128, 32, nengo.RectifiedLinear()),
-            True,
-            64,
-            0.55,
-            0.75,
-        ),
-        (benchmarks.integrator(128, 32, nengo.LIF()), True, 64, 1.0, 1.2),
+        (benchmarks.integrator(128, 32, nengo.RectifiedLinear()), True, 64, 0.5, 0.75,),
```

Comment:
I'd probably bump the 0.75 up to 0.8, it's the one that seems to fail most often.

Reply:
Yeah, I'm just reluctant to increase these margins any further. It's already (potentially) accepting a 50% performance decrease without complaint, which seems like a lot.

Another thing we could do is increase the number of iterations we use in this function. That should decrease the noise in these measurements, but it'd slow down the build. But probably more efficient than having to restart the build all the time. I'll try upping it to 10 and rerunning a few times and see what the failure rate looks like.

Reply:
Sounds good :+1:

Reply:
Restarted it a bunch, and it seems more stable (although still occasional failures). It didn't end up affecting the test time as much as I thought, since a good deal of the time is in the initial setup phase (which isn't affected by increasing the number of reps). So I put it up to 15 (which increases the test time by about 1 min total).

Reply:
Works for me :+1: